src/net.md

```diff
@@ -537,6 +538,75 @@ fn run() -> Result<()> {
 quick_main!(run);
 ```
 
+[ex-random-port-tcp]: #ex-random-port-tcp
+<a name="ex-random-port-tcp"></a>
+## Listen on Unused port TCP/IP
+
+[![std-badge]][std] [![cat-net-badge]][cat-net]
+
+Give the OS the responsibility to pick an unused [registered port].  These ports
+generally do not require sudo permission to bind to, and can be helpful when
+establishing connections with clients in a disposable communication chain.
+These ports begin with 1024 and have around 48,000 available.  Typically, a
+known port is used to initiate the communication and subsequent communication
+occurs on the registered port after a handshake process.
+
+In this example, the port is displayed on the console, and will listen until a
+request is made.  If you use your browser to test this program, simply enter
+the address:port into your location bar and make the request.  Because the
+program returns nothing, the browser's stop feature can be used to speed things
```

Comment:
This paragraph is overly word and redundant with later telnet testing one. 
I would remove all of it except the first sentence.